reporting/spacewalk-reports.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Oct 27 17:29:16 MSK 2020 - vzhestkov@suse.com
```

Comment:
No headers please as this will be added by our tooling when we submit the changes to the buildservice.

Reply:
@mcalmer , fixed. Thanks